submit/workflow/conditions.py

```diff
@@ -0,0 +1,68 @@
+from arxiv.submission.domain import Submission
```

Comment:
I split the conditions on the Workflow stages out for reuse and testing.

Reply:
Seems reasonable.